.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Description of Problem**:
+Short overview of the current situation.
```

Comment:
should we make these lines comments, I guess they will show up right?

Reply:
yes very good point